lib/modules/betteReddit.js

```diff
@@ -200,6 +205,9 @@ addModule('betteReddit', function(module, moduleID) {
 				default:
 					break;
 			}
+			if (module.options.truncateLongLinks.value) {
+				RESUtils.addCSS('overflow: hidden;text-overflow: ellipsis;white-space: nowrap;');
```

Comment:
@jkcchan 

Just remove the boolean check. A body class is added to the page automatically when options are on. So you just remove the oolean check and add the following to the scss file.

    .res-betteReddit-truncateLongLinks {
        .link .title {
            overflow: hidden;
            text-overflow: ellipsis;
            white-space: no-wrap;
        }
    }